api/envoy/extensions/filters/http/gcp_authn/v3/gcp_authn.proto

```diff
@@ -26,8 +26,7 @@ message GcpAuthnFilterConfig {
   // The URL format is "http://metadata.google.internal/computeMetadata/v1/instance/service-accounts/default/identity?audience=[AUDIENCE]"
   config.core.v3.HttpUri http_uri = 1 [(validate.rules).message = {required: true}];
 
-  // Retry policy for fetching tokens.
-  // This field is optional. If it is not configured, the filter will be fail-closed (i.e., reject the requests).
+  // Retry policy for fetching tokens. This field is optional.
```

Comment:
https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/gcp_authn_filter would also need to be updated

Reply:
Good catch! Done.